packages/components/package.json

```diff
@@ -69,6 +69,10 @@
 		"reakit": "^1.3.8",
 		"uuid": "^8.3.0"
 	},
+	"devDependencies": {
+		"@jest/globals": "27.5.1",
+		"@testing-library/react": "11.2.2"
+	},
```

Comment:
It looks like I can't list `devDependencies` in a package's `package.json`, but at the same time I have to list them in this file if I want to import them explicitly (e.g. importing `@jest/globals` in the unit test file).

What's the best solution here? Moving them to the root package.json in the `devDependencies` and then listing them again in this file's `dependencies`?

Reply:
I don't have a definitive answer as I'm not familiar with how deps are set up now.

I can say we shouldn't list these as direct `dependencies` of this package, that would cause consumers to download them when they're absolutely not necessary to _use_ this package.